string package_path         # path to ROS package containing controller module
string module_name          # name of the controller module within that package
string class_name           # controller class name within that module
string controller_name      # path to controller config parameters on param server
---
bool success
string reason
